test_conformance/relationals/test_comparisons_fp.h

```diff
@@ -0,0 +1,254 @@
+//
+// Copyright (c) 2022 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef _TEST_COMPARISONS_FP_H
+#define _TEST_COMPARISONS_FP_H
+
+#include <vector>
+#include <map>
+#include <memory>
+#include <CL/cl_half.h>
+
+#include "testBase.h"
+
+#define HALF_NAN 0x7e00
+
+//--------------------------------------------------------------------------/
```

Comment:
Do you mean any type of comment-based separators ?

Reply:
Yes; I am not aware of any coding style/guideline that recommends them, and we don't use them in the majority of the CTS code.

(also applies to `test_comparisons_fp.cpp` btw)

Reply:
>Yes; I am not aware of any coding style/guideline that recommends them, and we don't use them in the majority of the CTS code.

Obviously I will adapt this PR but here is my concern. Comment-based separators were already used in other CTS source files so I assumed it is not forbidden. Moreover, check-format.sh script does not point out such practice. In other words your comment introduces two problems:

- preventing such coding style automatically in the future
- removing existing cases in CTS

Do you think we should have a new issues to address those concerns ?

Reply:
The CTS code has a long history and different styles were followed, so the current state is far from perfect.  Removing existing cases in the CTS would be nice, but I wouldn't want that work to get in the way of functional improvements.

Not sure if we can easily automate detection of excess separators (and whether it's worth investing much time in it).  We may just have to rely on code reviewers to catch such style deviations.

Perhaps having a new issue is the best way forward indeed: there we could capture any style issues that aren't handled by check-format (as another example of a non-enforced style issue, there are lots of excess blank lines in `test_conformance/printf/util_printf.cpp`).

Reply:
>Perhaps having a new issue is the best way forward indeed

I think it is worth to point out missing coding guidelines doc which could help to avoid similar confusion in future

Corrections applied